[2023-12-27T17:49:02.780+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: modeling_dag.predict_ntu_task scheduled__2023-12-27T17:48:00+00:00 [queued]>
[2023-12-27T17:49:02.786+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: modeling_dag.predict_ntu_task scheduled__2023-12-27T17:48:00+00:00 [queued]>
[2023-12-27T17:49:02.786+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-12-27T17:49:02.797+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): predict_ntu_task> on 2023-12-27 17:48:00+00:00
[2023-12-27T17:49:02.803+0000] {standard_task_runner.py:57} INFO - Started process 3715 to run task
[2023-12-27T17:49:02.806+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'modeling_dag', 'predict_ntu_task', 'scheduled__2023-12-27T17:48:00+00:00', '--job-id', '675', '--raw', '--subdir', 'DAGS_FOLDER/modeling.py', '--cfg-path', '/tmp/tmpr8g2z1wk']
[2023-12-27T17:49:02.808+0000] {standard_task_runner.py:85} INFO - Job 675: Subtask predict_ntu_task
[2023-12-27T17:49:02.842+0000] {task_command.py:410} INFO - Running <TaskInstance: modeling_dag.predict_ntu_task scheduled__2023-12-27T17:48:00+00:00 [running]> on host c56b1de7f6cf
[2023-12-27T17:49:02.906+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='modeling_dag' AIRFLOW_CTX_TASK_ID='predict_ntu_task' AIRFLOW_CTX_EXECUTION_DATE='2023-12-27T17:48:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-27T17:48:00+00:00'
[2023-12-27T17:49:10.239+0000] {warnings.py:110} WARNING - /opt/***/dags/modeling.py:40: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  lstm_df[col] = low_val

[2023-12-27T17:49:10.743+0000] {logging_mixin.py:150} INFO - new_ntu_data [[34.609806], [36.33374], [32.972884]]
[2023-12-27T17:49:10.744+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/base.py:451: UserWarning: X does not have valid feature names, but MinMaxScaler was fitted with feature names
  "X does not have valid feature names, but"

[2023-12-27T17:49:10.744+0000] {logging_mixin.py:150} INFO - ss_ntu_data [[0.09688684]
 [0.1008646 ]
 [0.09310985]]
[2023-12-27T17:49:11.349+0000] {logging_mixin.py:150} INFO - 1/1 [==============================] - ETA: 0s
[2023-12-27T17:49:11.350+0000] {logging_mixin.py:150} INFO - 1/1 [==============================] - 1s 577ms/step
[2023-12-27T17:49:11.351+0000] {logging_mixin.py:150} INFO - new data : [[34.609806], [36.33374], [32.972884]]
[2023-12-27T17:49:11.352+0000] {logging_mixin.py:150} INFO - t+1 ntu : [31.37283]
[2023-12-27T17:49:11.352+0000] {logging_mixin.py:150} INFO - <class 'numpy.ndarray'>
[2023-12-27T17:49:11.356+0000] {python.py:183} INFO - Done. Returned value was: [31.37283]
[2023-12-27T17:49:11.372+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=modeling_dag, task_id=predict_ntu_task, execution_date=20231227T174800, start_date=20231227T174902, end_date=20231227T174911
[2023-12-27T17:49:11.403+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-12-27T17:49:11.421+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
